brave/browser/api/brave_api_component_updater.cc

```diff
@@ -142,6 +142,9 @@ void ComponentUpdater::RegisterComponent(const std::string& component_id) {
   } else if (component_id == kHoneyId) {
     RegisterComponentForUpdate(
         kHoneyPublicKeyStr, registered_callback, ready_callback);
+  } else if (component_id == kPinterestId) {
```

Comment:
@bbondy do we still need these? Can't we just register them in browser-laptop using the component api?